decayers/ERElasticScattering.cxx

```diff
@@ -215,30 +240,31 @@ Bool_t ERElasticScattering::Stepping() {
 }
 
 void  ERElasticScattering::ThetaRangesLab2CM(Double_t pM, Double_t tM) {
+  LOG(DEBUG) << "ERElasticScattering::ThetaRangesLab2CM(pM = " << pM << ", tM = " << tM << ")" << FairLogger::endl;
   Double_t rAng = fThetaRangeCenter*DegToRad();
   Double_t ratio = pM/tM;
   Double_t ratio2 = ratio*ratio;
-  Double_t rdTheta = fThetaRangedTheta*TMath::DegToRad(); // Detectors rdTheta
+  Double_t rdTheta = fThetaRangedTheta*DegToRad(); // Detectors rdTheta
   if (fRegisterIonStatus == kPROJECTILE) {
     // Projectile Ion
     if (pM != tM) {
```

Comment:
Please, explain, what does this condition mean?

Reply:
This condition means that there are formulas inside this if(pM != tM) that work only for cases with pM != tM.
For the cases with pM =tM other formulas are used.